Sources/MongoSwift/MongoCollection+ChangeStreams.swift

```diff
@@ -0,0 +1,102 @@
+import mongoc
+
+extension MongoCollection {
+    /**
+     * Starts a `ChangeStream` on a collection. The `CollectionType` will be associated with the
+     * `fullDocument` field in `ChangeStreamDocument`s emitted by the returned `ChangeStream`.
+     * Excludes system collections.
```

Comment:
What happens if I call `watch` on a system collection?

Reply:
It throws a `commandError`. Should we explicitly say a `commandError` is thrown when a system collection is used or doesn't the sentence`ServerError.commandError if an error occurs on the server while creating the change stream.`  already cover it?

Reply:
Yeah I think it does. Perhaps just rewording "Excludes ..." to "The server will return an error if this is called on a system collection." would be more informative.